scvi/data/_manager.py

```diff
@@ -455,20 +466,47 @@ def view_registry(self, hide_state_registries: bool = False) -> None:
         hide_state_registries
             If True, prints a shortened summary without details of each state registry.
         """
-        version = self._registry[_constants._SCVI_VERSION_KEY]
-        rich.print(f"Anndata setup with scvi-tools version {version}.")
-        rich.print()
-        self.view_setup_method_args(self._registry)
-
-        in_colab = "google.colab" in sys.modules
-        force_jupyter = None if not in_colab else True
-        console = rich.console.Console(force_jupyter=force_jupyter)
-        console.print(self._view_summary_stats())
-        console.print(self._view_data_registry())
+        console = AnnDataManager.view_registry_from_dict(self._registry)
 
         if not hide_state_registries:
             for field in self.fields:
                 state_registry = self.get_state_registry(field.registry_key)
                 t = field.view_state_registry(state_registry)
                 if t is not None:
                     console.print(t)
+
+    @staticmethod
+    def view_registry_from_dict(
```

Comment:
should this be public?

Reply:
I think it can be useful to users who want to print the registry w/o loading the model